.github/workflows/build-and-test.yml

```diff
@@ -193,6 +193,7 @@ jobs:
             sccache-path: /home/runner/.cache/sccache
     env:
       SCCACHE_DIR: ${{ matrix.sccache-path }}
+      CARGO_INCREMENTAL: 0 # disabled to reduce target cache size
```

Comment:
In my understanding this makes the cache smaller, but also decreases its usefulness, is that correct?

Reply:
Potentially. I've seen some reports of incremental compilation slowing builds too, however I did not notice much difference locally.

The main point is that the `wasm` target cache is the largest by far at ~1173 MB, compared to ~300 MB for the others. So any even minor reduction in size would be nice.
https://github.com/iotaledger/identity.rs/runs/3568663806?check_suite_focus=true

Reply:
Oh yeah, that's totally worth it!